var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -188,6 +190,13 @@ class Dd4hep(CMakePackage):
     )
     conflicts("~ddrec+dddetectors", msg="Need to enable +ddrec to build +dddetectors.")
 
+    @property
+    def libs(self):
+        # We need to override libs here, because we don't build a libdd4hep so
+        # the default discovery fails. All libraries that are built by DD4hep
+        # start with libDD
+        return fs.find_libraries("libDD*", root=self.prefix, shared=True, recursive=True)
```

Comment:
`fs` not needed, see e.g. `libxinerama` link in #39517.
```suggestion
        return find_libraries("libDD*", root=self.prefix, shared=True, recursive=True)
```

Reply:
Tested this again without the explicit import and I get a 

```
==> Error: AttributeError: Query of package 'dd4hep' for 'libs' failed
	prefix : /home/tmadlener/work/.spack/spackages/dd4hep/master/skylake-ubuntu22.04-gcc12.3.0/jkb6ous
[....]
```

Putting it back in.

Reply:
I think that's maybe because this uses
```
        env.prepend_path("LD_LIBRARY_PATH", self.spec["dd4hep"].libs.directories[0])
```
instead of
```
        env.prepend_path("LD_LIBRARY_PATH", self.libs.directories[0])
```

I don't have the `fs` import and it installs without the attribute error.

Reply:
You are correct. Updated the PR. Thanks for the pointer.